src/pages/GetStarted/GetStarted.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import { NavButton } from 'components/NavButton/NavButton';
-import { InstructionModal } from 'components/InstructionModal/InstructionModal';
+import { Modal as InstructionModal } from 'components/Modal/Modal';
 import backgroundArrow from '../../assets/background-arrow-grey.svg';
 import './GetStarted.css';
+import { ModalType } from 'canvas/Enums';
 
 const button: NavButton = {label: "GET STARTED", to: "/AddPhoto" };
-const intro: InstructModal = {text: ["Let’s Make a \npattern in 3 \nsteps"]};
-const steps: InstructModal = {text: ["1. Upload photo", "2. Trace pattern", "3. Print pdf"]};
+const intro: Modal = {text: ["Let’s Make a \npattern in 3 \nsteps"], type: ModalType.Instruction};
+const steps: Modal = {text: ["1. Upload photo", "2. Trace pattern", "3. Print pdf"], type: ModalType.Instruction};
```

Comment:
Not a big deal, but elsewhere I think we tended to put these inside the component definition.

Reply:
I will move them in.